frontend/src/Pages/Home/InputForm/InputForm.tsx

```diff
@@ -13,8 +13,10 @@ import {
   Select,
```

Comment:
looks like the input form in this PR is using an old version. Please check the files and what you're changing in your PR.

Reply:
updated